acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -0,0 +1,39 @@
+require 'puppet/acceptance/common_utils'
```

Comment:
@james-stocks I added the needed install code to `setup/aio/pre-suite/010_Install.rb` and removed `/setup/aio/010_Install.rb`. Seemed undesirable to add it to `setup/common/020_Configure_Pxp_Agents.rb` since it is AIO specific.